packages/pigeon/lib/generator_tools.dart

```diff
@@ -82,3 +82,37 @@ class Indent {
 String makeChannelName(Api api, Method func) {
   return 'dev.flutter.dartle.${api.name}.${func.name}';
 }
+
+/// Represents the mapping of a Dart datatype to a Host datatype.
+class HostDatatype {
+  /// Parametric constructor for HostDatatype.
+  HostDatatype({this.datatype, this.isBuiltin});
+
+  /// The [String] that can be printed into host code to represent the type.
+  final String datatype;
+
+  /// `true` if the host datatype is something builtin.
```

Comment:
Still not sure what this means. Does this mean primitive to Java? Or a Dart SDK class in Dart?

Reply:
ohhh wait, this is the builtin handled types for our codec?

Reply:
Right, as opposed to types defined as collections of supported types.  These are the atomic types.